source/enable-autocomplete.txt

```diff
@@ -0,0 +1,98 @@
+.. _vs-code-enable-autocomplete:
+
+====================
+Enable Auto Complete
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can enable autocomplete for MongoDB Playgrounds. Enabling 
+autocomplete allows you to automatically populate database and collection 
+names when using ``use("")`` or 
+``db.getCollection("")`` syntax within a MongoDB playground.
+
+Before you Begin
+----------------
+
+Install the :ref:`MongoDB VSCode extension <vsce-install>`.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Open Settings.json
+     
+      In VSCode, open the command palette:
+
+      .. list-table::
+         :header-rows: 1
+         :widths: 30 70
+
+         * - Operating System
+           - Keyboard Command
+
+         * - Windows
+           - :kbd:`CTRL` + :kbd:`SHIFT` + :kbd:`P`
+
+         * - MacOS
+           - :kbd:`CMD` + :kbd:`SHIFT` + :kbd:`P`
+
+      Enter :guilabel:`Open User Settings (JSON)` and select the option 
+      from the autocompleted list:
+
+      .. figure:: /images/vsce-command-palette-json-settings.png
+         :figwidth: 700px
+         :alt: Image of a stream connection
+
+   .. step:: Add the autocomplete snippet to the ``Settings.json`` file
+     
+      Copy the following code snippet:
+
+      .. code-block::
+
+         "editor.quickSuggestions": {
+            "other": true,
+            "comments": false,
+            "strings": true
+         }
+
+      Paste the code into the ``Settings.json`` file:
+
+      .. figure:: /images/vsce-add-snippet-json-settings.png
+         :figwidth: 700px
+         :alt: Image of a stream connection
+
+   .. step:: Test the autocompletion feature
+
+      In a new playground file connected to your deployment, 
+      type the first two characters of a collection name between  
+      ``db.getCollection("")`` and click the full name to populate 
+      the collection name:
+
+      .. figure:: /images/vsce-get-collection-auto-complete.png
+         :figwidth: 700px
+         :alt: Get collection autocomplete
+
+      You can also autocomplete database names by typing the first 
+      character of a database name in-between the ``use("")`` statement:
+
+      .. figure:: /images/vsce-use-database-auto-complete.png
+         :figwidth: 700px
+         :alt: Use database autocomplete
+
+      .. tip::
+
+         You can also press :kbd:`Ctrl` + :kbd:`Space` on your keyboard 
+         to autocomplete.
```

Comment:
I wasn't able to get this to work. I'm trying to press ctrl + space here:

<img width="577" alt="Screenshot 2024-04-08 at 11 34 41 AM" src="https://github.com/mongodb/docs-mongodb-vscode/assets/29379027/9441ea77-e7fe-4d04-8e3a-e9cf214b8c36">

Reply:
Yeah, this was working for me Friday, but does not seem to be working for me today. I will check with Alena in TR to see whats up, I thought it may have been my environment only.